go/consensus/tendermint/abci/mux.go

```diff
@@ -679,11 +680,14 @@ func (mux *abciMux) EstimateGas(caller signature.PublicKey, tx *transaction.Tran
 	}
 	_ = tx.Fee.Amount.FromUint64(math.MaxUint64)
 
-	ctx.SetTxSigner(caller)
+	ctx.SetTxAccount(caller)
 	mockSignedTx := transaction.SignedTransaction{
-		Signed: signature.Signed{
-			Blob: cbor.Marshal(tx),
-			// Signature is fixed-size, so we can leave it as default.
+		Envelope: v0.Envelope{
+			Account: *caller,
+			Signatures: []*signature.RawSignature{
```

Comment:
So this always assumes a single signer, so EstimateGas will underestimate for multiple signatures. I guess we can fix this later.

Reply:
It now assumes everyone signs the tx.